docs/PlantUML/CommandComponent.puml

```diff
@@ -0,0 +1,26 @@
+@startuml
+
```

Comment:
A common problem among all groups including mine is to have the little icons. You can add:
```
hide circle
skinparam classAttributeIconSize 0
```
to remove them and follow the recommended format.